CMakeLists.txt

```diff
@@ -396,6 +397,10 @@ endif ()
 if (ENABLE_SCRIPTING)
     target_compile_options(libopenrct2 PUBLIC -DENABLE_SCRIPTING)
 endif ()
+if (ENABLE_ASAN)
+    target_compile_options(libopenrct2 PUBLIC -fsanitize=address)
+    target_link_options(libopenrct2 PUBLIC -fsanitize=address)
```

Comment:
Should this affect the cli and ui targets as well?

Reply:
It does since `PUBLIC` is specified.